dotcom-rendering/src/web/components/TableOfContents.tsx

```diff
@@ -0,0 +1,88 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { line, neutral, space, textSans } from '@guardian/source-foundations';
+import {
+	SvgChevronDownSingle,
+	SvgChevronUpSingle,
+} from '@guardian/source-react-components';
+import type { TableOfContentsItem } from 'src/types/frontend';
+
+interface Props {
+	tableOfContents: TableOfContentsItem[];
+}
+
+const anchorStyles: SerializedStyles = css`
+	color: black;
+	text-decoration: none;
+`;
+
+const listItemStyles: SerializedStyles = css`
+	${textSans.xsmall({ fontWeight: 'bold', lineHeight: 'regular' })}
+	border-bottom: 1px solid ${line.primary};
+	padding: 6px 0;
+`;
+
+const detailsStyles: SerializedStyles = css`
+	padding: 6px 0;
+	&:not([open]) .is-on,
+	&[open] .is-off {
+		display: none;
+	}
+	summary::-webkit-details-marker {
+		display: none;
+	}
```

Comment:
Is this browser prefix needed? I think we handle these automatically

Reply:
Ah, it is removing the triangle!

Reply:
added comment